scripts/tioga.flux.sh

```diff
@@ -20,7 +20,7 @@ export PYOPENCL_CTX="AMD:0"
 
 run_cmd="flux run -N $nnodes -n $nproc --exclusive"
 
-export XDG_CACHE_ROOT="/tmp/$USER/xdg-scratch"
-export POCL_CACHE_ROOT="/tmp/$USER/pocl-cache"
+export XDG_CACHE_ROOT=${XDG_CACHE_HOME:-"$(pwd)/xdg-scratch"}
```

Comment:
Should we make `*-scratch` or `*-cache` the standard directory? Currently it's 50-50.

Reply:
I like cache myself, but meh.

Reply:
renamed them to -cache in [92e227a](https://github.com/illinois-ceesd/mirgecom/pull/1017/commits/92e227a2a7aac2bd8e4c61495ec64626d518ce7d)